docs/api/scheduler.md

```diff
@@ -31,6 +31,9 @@ Many of these APIs return collections of resources. Internally, all resources ar
 are returned in units of bytes while resources of type `vcore` are returned in units of millicores
 (thousands of a core). All other resource types have no specific unit assigned.
 
+Additionally, it should be note that in the JSON response bodies, some fields may be omitted if their values are empty. Specifically, if a field is not present in the response, it can be assumed that the value is empty/null.
```

Comment:
We can use the following markup type to show this in a nicer way:
```
:::note PUT TEXT HERE :::
```
That would allow the removal of: "Additionally, it should be note that"
See https://docusaurus.io/docs/markdown-features/admonitions